src/v0/destinations/snapchat_custom_audience/utils.js

```diff
@@ -34,9 +37,7 @@ const validatePayload = message => {
 const validateFields = (schema, data) => {
   // if required field is not present in all the cases
   if (data[0].length === 0) {
-    throw new InstrumentationError(
-      `${schema} is required for the chosen schema`
-    );
+    throw new ConfigurationError(`${schema} is required for the chosen schema`);
```

Comment:
Somehow this error message doesn't make sense. Can you fix it?

Reply:
i made change in sentence, can you confirm it?